In [1]:
import pandas as pd

In [3]:
data = pd.read_csv('../data/apartments.csv')

In [7]:
data.head(2).T

0  \
_id                                                61726ccfd9b0bc0e40542c46   
added_at                                                         2022-11-05   
address.city                                                         Astana   
address.corner_street                                               №_23-16   
address.country                                                   Kazahstan   
address.district                                                 Almaty_r-n   
address.house_num                                                       NaN   
address.street                                         Temirbeka_Zhurgenova   
archived_at                          Будет продлено через <b>2&nbsp;дня</b>   
area_with_kitchen                              67 м², Площадь кухни — 17 м²   
balcony                                       несколько балконов или лоджий   
bathroom_count                                                  совмещенный   
ceiling                                                               2.8 м   
condition                                                               NaN   
created_at                                         2021-08-27T00:00:00.000Z   
deleted_at                                   Будет удалено 5 ноября 2024 г.   
description               Комнаты изолированы, встроенная кухня, новая с...   
door_type                                                     бронированная   
floor                                                                8 из 9   
floor_type                                                           паркет   
full_address              Астана, Алматы р-н, Темирбека Жургенова — № 23-16   
furniture                                                         полностью   
geo_location.coordinates                               [71.50252,51.121004]   
geo_location.type                                                     Point   
house_type                                                        кирпичный   
is_dorm                                                                 нет   
is_migrated                                                            True   
lat                                                               51.121004   
location                                                                NaN   
lon                                                                71.50252   
material                                                          кирпичный   
owner.isBuilder                                                       False   
owner.isChecked                                                       False   
owner.isComplex                                                       False   
owner.isCurrentUser                                                   False   
owner.isOwner                                                          True   
owner.isPro                                                           False   
owner.label                                                             NaN   
owner.title                                                          Хозяин   
parking                                                             паркинг   
parsed_at                                          2022-11-10T00:00:00.000Z   
price                                                              29400000   
price_history             [{"price":25800000,"parsed_at":{"$date":"2021-...   
residential_complex                                                Сатурн 2   
rooms                                                                     2   
security                  охрана, домофон, кодовый замок, видеонаблюдени...   
square                                                                 67.0   
url                                      https://krisha.kz/a/show/669773388   
uuid                                   8da6a192-1df5-49e3-9bc5-e481b511daf3   
year                                                                   2022   

                                            

In [5]:
len(data)

39422

The initial dataset contains **39422 samples** with **50 columns of features**. There are unuseful features such as *country* of apartment or *full_address*, etc. We do not need these columns, the reason is we have geolocation column which is combination of *longitude* and *latitude*.